<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgLensBlur</class>
 <widget class="QWidget" name="WdgLensBlur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>172</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Iris</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Shape:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="irisShapeCombo">
        <item>
         <property name="text">
          <string>Triangle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quadrilateral (4)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pentagon (5)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hexagon (6)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Heptagon (7)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Octagon (8)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Radius:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KisIntParseSpinBox" name="irisRadiusSlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="label" stdset="0">
         <string>Radius:</string>
        </property>
        <property name="sliderEnabled" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Rotation:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KisAngleSelector" name="irisRotationSelector" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisIntParseSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_int_parse_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>KisAngleSelector</class>
   <extends>QWidget</extends>
   <header location="global">KisAngleSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>irisShapeCombo</tabstop>
  <tabstop>irisRadiusSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
